license_manager/service/service_test.go

```diff
@@ -0,0 +1,59 @@
+package service
+
+import (
+	"context"
+	"testing"
+
+	lmpb "github.com/enfabrica/enkit/license_manager/proto"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func TestAllocateUnimplemented(t *testing.T) {
+	ctx := context.Background()
+	s := &Service{}
+	wantCode := codes.Unimplemented
+	req := &lmpb.AllocateRequest{}
+
+	_, err := s.Allocate(ctx, req)
+	if gotCode := status.Code(err); gotCode != wantCode {
+		t.Errorf("got code %v; want code %v", gotCode, wantCode)
```

Comment:
nit: don't feel strongly, but we've been using the assert library within the repo. I know there are differing opinions related to its use, but recommend using it for simple tests even if just for consistency.

Reply:
Done! It is much cleaner; thanks :)